testsuite/features/build_validation/create_bootstrap_repositories/srv_create_bootstrap_repositories.feature

```diff
@@ -14,122 +14,66 @@ Feature: Create bootstrap repositories
   Scenario: Create the bootstrap repository for a SLES 12 SP4 minion
     When I create the bootstrap repository for "sle12sp4_minion" on the server
 
-@sle12sp4_ssh_minion
-  Scenario: Create the bootstrap repository for a SLES 12 SP4 Salt SSH minion
-    When I create the bootstrap repository for "sle12sp4_minion" on the server
-
```

Comment:
Good catch

Reply:
As a matter of fact, this was not removed because it was wrong, but because it was useless.